working/0546-patterns/patterns-feature-specification.md

```diff
@@ -1836,8 +2049,134 @@ main() {
 
 *This prints "1", "2", "here".*
 
+If a variable has already been explicitly initialized before the initializer
+ran and any destructuring occurred, then it keeps its previously initialized
+value:
+
+```dart
+main() {
+  late var (a, b) = (1, 2);
+  a = 3;
```

Comment:
Well, that's awkward!

Consider *very strongly* not to allow late patterns! Just to avoid this!

Reply:
Agreed!